src/Main.js

```diff
@@ -0,0 +1,28 @@
+import {Component} from 'react';
+import name from images
+import name from images
+
+class Main extends Component {
+  render () {
+    return (
+      <>
+        <HornedBeast title={'Name'} imgURL={name} decriptio={'Descriptive block'} />
+        <HornedBeast title={'Scratchy'}
+      </>
```

Comment:
Nice work so far! 

Looks like this poor `<HornedBeast>` needs it's `imageURL` and `description` as well 👍

Reply:
Hello there!
  I realized I got ahead of myself here, my apologies. For this lab, you could pass in whatever title, imgUrl and description you'd like, but it should be a working description and title etc. 👍